services/search/search_engine.py

```diff
@@ -20,7 +24,39 @@ def __init__(self, articles, keywords_weight=1, synonyms_weight=1):
         self.keywords_weight = keywords_weight
         self.synonyms_weight = synonyms_weight
 
-    def query(self, keywords, synonyms):
+    def query(self, query):
+        """
+        Extracts keywords, from keywords endpoint, 
+        and synonyms, from local function,
+        to call search query and calculate the score for articles
+        Args:
+            query: query string
+        Returns:
+            A map of the score of every article
+        """
+        url = "http://localhost:8080"
```

Comment:
Can you use the `KEYWORDS_ENDPOINT` in the constants file here?

Reply:
HAHAHHA silly me